src/screens/ShotsScreen.tsx

```diff
@@ -1,24 +1,46 @@
 import React from "react";
-import { View, StyleSheet, Text } from "react-native";
+import { NavigationProp, Route } from "@react-navigation/native";
+import { View, StyleSheet, Text, FlatList } from "react-native";
+import PhotoGalleryView from "../components/photos/PhotoGalleryView";
+import { getPhotosForUser, Photo } from "../providers/PhotoProvider";
```

Comment:
`Photo` doesn't seem to be used and shouldn't be imported.
```suggestion
import { getPhotosForUser } from "../providers/PhotoProvider";
```